data-gen/snoopAuditlog.py

```diff
@@ -139,11 +191,11 @@ def generate_coverage_report(openapi_spec, audit_log):
         if operationId not in test_tags[tag]:
           test_tags[tag].append(operationId)
 
-      agent = useragent.split(' ')[0]
-      if agent not in useragents.keys():
-        useragents[agent] = []
-      if operationId not in useragents[agent]:
-        useragents[agent].append(operationId)
+    agent = useragent.split(' ')[0]
+    if agent not in useragents.keys():
+      useragents[agent] = []
+    if operationId not in useragents[agent]:
+      useragents[agent].append(operationId)
```

Comment:
This indention (too far in) is why we lost useragents other than e2e... fixed now.
I'm not so sure about this 2 char indention.